cardano-cli/src/Cardano/CLI/Shelley/Run/Governance.hs

```diff
@@ -73,7 +73,7 @@ runGovernanceMIRCertificate mirPot sAddrs rwdAmts (OutputFile oFp) = do
                oFp (length sAddrs) (length rwdAmts)
 
     let sCreds  = map stakeAddrToStakeCredential sAddrs
-        mirCert = makeMIRCertificate mirPot (zip sCreds rwdAmts)
+        mirCert = makeMIRCertificate mirPot (StakeAddressesMIR $ zip sCreds rwdAmts)
```

Comment:
And another TODO for the cli here for a follow-up PR is to expose the new MIR functionality.

Reply:
I added a TODO comment